docs/operator/helm-getting-started.md

```diff
@@ -0,0 +1,477 @@
++++
+title = "Grafana Agent Operator Helm Quickstart"
+weight = 110
++++
+# Grafana Agent Operator Helm Quickstart
+
+In this guide you'll learn how to deploy the [Grafana Agent Operator]({{< relref "./_index.md" >}}) into your Kubernetes cluster using the [grafana-agent-operator Helm chart](https://github.com/grafana/helm-charts/tree/main/charts/agent-operator). 
+
+You'll then deploy the following custom resources (CRs):
+
+- A `GrafanaAgent` resource, which discovers one or more `MetricsInstance` and `LogsInstances` resources.
+- A `MetricsInstance` resource that defines where to ship collected metrics. Under the hood, this rolls out a Grafana Agent StatefulSet that will scrape and ship metrics to a `remote_write` endpoint.
+- A `ServiceMonitor` resource to collect cAdvisor and kubelet metrics. Under the hood, this configures the `MetricsInstance` / Agent StatefulSet.
+- A `LogsInstance` resource that defines where to ship collected logs. Under the hood, this rolls out a Grafana Agent DaemonSet that will tail log files on your cluster nodes.
+- A `PodLogs` resource to collect container logs from Kubernetes Pods. Under the hood, this configures the`LogsInstance` / Agent DaemonSet.
+
+To learn more about the custom resources Operator provides, please consult [Operator architecture]({{< relref "./architecture.md" >}}).
+
+> **Note:** Agent Operator is currently in beta and its custom resources are subject to change as the project evolves. It current supports the metrics and logs subsystems of Grafana Agent. Integrations and traces support is coming soon.
+
+By the end of this guide, you'll have deloyed Agent Operator into your cluster and will be scraping and shipping cAdvisor and kubelet metrics to a Prometheus-compatible metrics endpoint. You'll also be collecting and shipping your Pods' container logs to a Loki-compatible logs endpoint.
+
+## Prerequisites
+
+Before you begin, make sure that you have the following available to you:
+
+- A Kubernetes cluster
+- The `kubectl` command-line client installed and configured on your machine
+- The `helm` command-line client installed and configured on your machine
+
+## Step 1: Install Agent Operator Helm Chart
+
+In this step you'll install the [grafana-agent-operator Helm chart](https://github.com/grafana/helm-charts/tree/main/charts/agent-operator) into your Kubernetes cluster. This will install the latest version of Agent Operator and its [Custom Resource Definitions](https://github.com/grafana/agent/tree/main/production/operator/crds) (CRDs). By default the chart will also deploy a Service that allows you scrape kubelets using a `ServiceMonitor`.
```

Comment:
```suggestion
In this step you'll install the [grafana-agent-operator Helm chart](https://github.com/grafana/helm-charts/tree/main/charts/agent-operator) into your Kubernetes cluster. This will install the latest version of Agent Operator and its [Custom Resource Definitions](https://github.com/grafana/agent/tree/main/production/operator/crds) (CRDs). By default the chart will configure the operator to maintain a Service that allows you scrape kubelets using a `ServiceMonitor`.
```